ui/src/app/kvm/components/KVMDetailsHeader/KVMDetailsHeader.tsx

```diff
@@ -22,7 +22,7 @@ type Props = {
   id: Pod["id"];
   headerContent: KVMHeaderContent | null;
   setHeaderContent: KVMSetHeaderContent;
-  setSearchFilter: SetSearchFilter;
+  setSearchFilter?: SetSearchFilter;
```

Comment:
This isn't required for `VirshDetails` since we can't perform machine actions from that page.